### ISSUE TITLE
Basic logging system

### ISSUE THREAD

**Issue by calvinrbnspiess** [kadirahq/meteor-debug#3]
Hey,
Kadira Debug is really impressive and helps a lot to debug Meteor apps.
But to replace the good old-known Developer Tools, I miss a basic logging system to debug easier states from variables (or whatever). I played around a bit with the package and found a way to insert custom events and format them with HTML, but that solution is not so good and a real logging integration would still improve Kadira.
A simple implementation of a logging system might include multiple states (such as information, warning and error) and should not take too much time to develop. I hope you like my idea and I am sorry for my bad English :/

**Comment by arunoda** [comment]
Okay. That means, you seems to like to have a custom logging functionality to Kadira Debug. We can simply track console log. But I prefer to our own API.

What do you think?

> BTW: Kadira Debug is not to replace Chrome Dev Tools or similar. But another tool to add into your existing Dev Tools.

**Comment by calvinrbnspiess** [comment]
Yes, I would also prefer a custom API. You mentioned it should not replace the Dev Tools, that's a good point. - But I think Kadira Debug actually implements the most needed tools and it would be great to have to use only one tool in the most time.

**Comment by arunoda** [comment]
Okay. Great. I'll add a logging API.
Right now, I'm planning for the launch and so, this will get bit delayed (end of this week like)

**Comment by calvinrbnspiess** [comment]
Ok that's great. Just take as much time as needed.
